### ISSUE TITLE
error on resizeFullScreen and saveScreenshot(fullPage: true)

### ISSUE THREAD

**Issue by d-a-n** [LucianoGanga/simple-headless-chrome#64]
When using `resizeFullScreen()` or `saveScreenshot()` with `fullPage: true` the following error is thrown:

```
ERROR! { Error: Invalid parameters (fitWindow: boolean value expected)
    at /node_modules/chrome-remote-interface/lib/chrome.js:90:30
    at Chrome.handleMessage (/node_modules/chrome-remote-interface/lib/chrome.js:289:13)
    at WebSocket.<anonymous> (/node_modules/chrome-remote-interface/lib/chrome.js:266:27)
    at emitTwo (events.js:125:13)
    at WebSocket.emit (events.js:213:7)
    at Receiver._receiver.onmessage (/node_modules/ws/lib/WebSocket.js:143:54)
    at Receiver.dataMessage (/node_modules/ws/lib/Receiver.js:385:14)
    at extension.decompress (//node_modules/ws/lib/Receiver.js:354:40)
    at _inflate.flush (/node_modules/ws/lib/PerMessageDeflate.js:279:12)
    at afterWrite (_stream_writable.js:452:3)
    at onwrite (_stream_writable.js:443:7)
    at InflateRaw.afterTransform (_stream_transform.js:90:3)
    at Zlib.callback (zlib.js:514:5)
  response: 
   { code: -32602,
     message: 'Invalid parameters',
     data: 'fitWindow: boolean value expected' } }
```

**Comment by fusionx1** [comment]
Same error im getting using the sample code.

**Comment by iyyappan13391** [comment]
Hi,
Try with boolean true to string 'true'.

**Comment by LucianoGanga** [comment]
Hello @d-a-n ! 

Can I know which version are you using? 

Try installing the last version (at this moment is v.4.3.7)

Thanks!
Luciano

**Comment by LucianoGanga** [comment]
Fixed on 4.3.9!  :)
